app/components/DepartureRow.tsx

```diff
@@ -7,34 +7,16 @@ function DepartureRow(props: any) {
     const {departure} = props    
 
     return(
-        <div className="departure-row">
+        departure
```

Comment:
So you should either do some conditionnal rendering within a container div like 
```
return (
<div>
{
departure
? ....
: ....
}
</div>
)

```
or with 2 different return like 
```
if (departure) {
return (<div>...</div>)
} else {
return (<div>...</div>)
}
```

but no conditionnal check just after `render`